src/auth/oidc/models/openid.class.spec.ts

```diff
@@ -1,5 +1,64 @@
 import oidc from './openid.class'
+import passport from 'passport'
+import { Request, Response, NextFunction } from 'express'
 
 test('OIDC Auth', () => {
     expect(oidc).toBeDefined()
 })
+
+test('OIDC configure serializeUser', () => {
```

Comment:
Could we also test the expected behaviour please? E.g whether an event gets emitted etc